openapi/v2/core-data.yaml

```diff
@@ -459,87 +459,59 @@ components:
       requestBody:
         required: true
         content:
           application/json:
             schema:
-              type: array
-              items:
-                $ref: '#/components/schemas/AddEventRequest'
+              $ref: '#/components/schemas/AddEventRequest'
             example:
-              - event:
-                  deviceName: device-002
-                  id: d5471d59-2810-419a-8744-18eb8fa03465
-                  origin: 1602168089665565300
-                  readings:
-                    - deviceName: device-002
-                      resourceName: resource-002
-                      profileName: profile-002
-                      id: 7003cacc-0e00-4676-977c-4e58b9612abd
-                      origin: 1602168089665565300
-                      valueType: Float32
-                      value: '12.2'
-              - event:
-                  deviceName: device-002
-                  id: d5471d59-2810-419a-8744-18eb8fa03465
-                  origin: 1602168089665565300
-                  readings:
-                    - deviceName: device-002
-                      resourceName: resource-002
-                      profileName: profile-002
-                      id: 7003cacc-0e00-4676-977c-4e58b9612abd
-                      origin: 1602168089665565300
-                      valueType: Float32
-                      value: '12.2'
-              - event:
-                  deviceName: device-002
-                  id: 73fc4f9c-2d64-4920-addb-b1f33a8f8514
-                  origin: 1602168089665565300
-                  readings:
-                    - apiVersion: v2
-                      created: 1594879337014
-                      deviceName: device-002
-                      resourceName: resource-002
-                      profileName: profile-002
-                      id: 71c601d9-cb56-453a-8c75-54461e444713
-                      origin: 1602168089665565300
-                      valueType: Binary
-                      binaryValue: '83010203'
-                      mediaType: image
+              event:
+                deviceName: device-002
+                profileName: profile-002
+                id: d5471d59-2810-419a-8744-18eb8fa03465
+                origin: 1602168089665565300
+                readings:
+                  - deviceName: device-002
+                    resourceName: resource-002
+                    profileName: profile-002
+                    id: 7003cacc-0e00-4676-977c-4e58b9612abd
+                    origin: 1602168089665565300
+                    valueType: Float32
+                    value: '12.2'
       responses:
-        '207':
-          description: "Indicates a multi-part response supportive of accepting multiple requests at once. The 'statusCode' property of each response in the returned array will indicate success or failure."
+        '201':
+          description: "Indicates the event has been successfully created."
```

Comment:
Should this be `event has been successfully added`?  since it my not be created it persistence is off.

Reply:
Updated.